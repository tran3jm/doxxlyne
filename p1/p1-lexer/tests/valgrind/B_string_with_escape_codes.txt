==26919== Memcheck, a memory error detector
==26919== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26919== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==26919== Command: ../decaf inputs/string_ex_codes.decaf
==26919== 
ID       [line 001]  str
SYMBOL   [line 001]  =
STRLIT   [line 001]  "erw\t\n"
==26919== 
==26919== HEAP SUMMARY:
==26919==     in use at exit: 1,600 bytes in 38 blocks
==26919==   total heap usage: 1,147 allocs, 1,109 frees, 144,132 bytes allocated
==26919== 
==26919== LEAK SUMMARY:
==26919==    definitely lost: 64 bytes in 1 blocks
==26919==    indirectly lost: 1,536 bytes in 37 blocks
==26919==      possibly lost: 0 bytes in 0 blocks
==26919==    still reachable: 0 bytes in 0 blocks
==26919==         suppressed: 0 bytes in 0 blocks
==26919== Rerun with --leak-check=full to see details of leaked memory
==26919== 
==26919== For lists of detected and suppressed errors, rerun with: -s
==26919== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
