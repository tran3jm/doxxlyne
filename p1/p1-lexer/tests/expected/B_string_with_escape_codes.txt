ID       [line 001]  str
SYMBOL   [line 001]  =
STRLIT   [line 001]  "erw\t\n"
